guides/common/modules/proc_creating-partition-tables.adoc

```diff
@@ -28,7 +28,7 @@ autopart
 You can also use the *Template* file browser to upload a template file.
 +
 The format of the layout must match that for the intended operating system.
-For example, {RHEL} 7.2 requires a layout that matches a kickstart file.
+For example, {RHEL} requires a layout that matches a kickstart file.
```

Comment:
```suggestion
For example, {EL} requires a layout that matches a kickstart file.
```
Is RHEL included on purpose or could  we generalize it to EL?

Reply:
Applied!